### PR TITLE
Add a basic tooltip showing who reacted

### PR DESCRIPTION
**Pull request by jryans**
2019-05-17 at 12 22

Also tweaks the edited tooltip to match:

2019-05-17 at 12 22

This does not match the designed tooltip, but it's a step along the way.

Part of https://github.com/vector-im/riot-web/issues/9722

### CODE REVIEW

File: res/css/views/messages/_ReactionsRowButtonTooltip.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    font-size: 8px;
+    padding: 6px;
+    border: none;
+    border-radius: 3px;
+
+    .mx_Tooltip_chevron::after {
+        border-right-color: $reaction-row-button-tooltip-bg-color;
+    }
+
+    .mx_ReactionsRowButtonTooltip_reactedWith {
```
Comment (bwindels) [inline res/css/views/messages/_ReactionsRowButtonTooltip.scss]: Does the design imply that this class and mx_ReactionsRowButtonTooltip_senders are inline? It looks a bit weird to me to have a line break in-between.

File: src/components/views/messages/TextualBody.js [hunk 2]
```diff
@@ -448,14 +448,17 @@ module.exports = React.createClass({
...
+                label={_t("Edited at %(date)s.", {date})}
+            />;
         }
         return (
             <div
                 key="editedMarker" className="mx_EventTile_edited"
                 onMouseEnter={this._onMouseEnterEditedMarker}
                 onMouseLeave={this._onMouseLeaveEditedMarker}
-            >{editedTooltip}<span>{`(${_t("Edited")})`}</span></div>
+            >{editedTooltip}<span>{`(${_t("edited")})`}</span></div>
```
Comment (bwindels) [inline src/components/views/messages/TextualBody.js]: thanks for this :)

File: res/css/views/messages/_ReactionsRowButtonTooltip.scss (hunk 1 above)
Comment (jryans) [reply res/css/views/messages/_ReactionsRowButtonTooltip.scss]: That's how I interpreted it, but I'll ask! 😁

Comment (jryans) [reply res/css/views/messages/_ReactionsRowButtonTooltip.scss]: Separate line confirmed!

**Comment by johanricher** [comment]
Just passing to say that I'm a simple user testing Riot with labs featured actived on https://riot.im/develop/ and I LOVE the new changes you're all working on. Believe me people will go crazy for them when it's out. Keep up the good work!

**Pull request merged**